docs/pages/application-access/troubleshooting-apps.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Troubleshooting Application Access
+description: Describes common issues and solutions for access to applications.
```

Comment:
```suggestion
description: Describes common issues and solutions for access to applications proxied by Teleport.
```
Or something else that makes it clear that these are applications protected by Teleport.